hardware-and-software-requirements.md

```diff
@@ -18,8 +18,8 @@ Starting from v6.1.1, TiDB provides multi-level support for different quality st
 
     |  Operating systems   |   Supported CPU architectures   |
     |   :---   |   :---   |
-    | Red Hat Enterprise Linux 8.4 or a later 8.x version  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
-    | CentOS 8 Stream | <ul><li>x86_64</li><li>ARM 64</li></ul> |
+    | <ul><li>Red Hat Enterprise Linux 8.4 or a later 8.x version</li><li>CentOS 8.4 or a later 8.x version</li></ul>  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
+    | Red Hat Enterprise Linux 8 Stream | ARM 64 |
```

Comment:
There is no "Stream" version of RHEL. Only CentOS has a "Stream"

CentOS Stream 8 is a rolling upgrade without minor versions.
RHEL 8 is not a rolling release and has minor versions.

See also https://access.redhat.com/support/policy/updates/errata

![image](https://user-images.githubusercontent.com/1272980/199415294-fa77eefb-cb37-4d6d-b968-03dcd50c0dbc.png)